apps/experiments/migrations/0075_alter_participant_unique_together_and_more.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.7 on 2024-04-09 12:56
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('teams', '0005_invitation_groups'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('experiments', '0074_participantdata'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='participant',
+            unique_together={('team', 'identifier')},
+        ),
+        migrations.AddField(
+            model_name='participant',
+            name='external_chat_id',
+            field=models.CharField(null=True),
+        ),
+        migrations.AddField(
+            model_name='participant',
+            name='user',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+        ),
+        migrations.AlterUniqueTogether(
+            name='participant',
+            unique_together={('team', 'identifier'), ('team', 'external_chat_id')},
+        ),
+    ]
```

Comment:
can merge these operations into the previous migration (or re-generate the previous migration). I was going to suggest using https://docs.djangoproject.com/en/5.0/ref/django-admin/#cmdoption-makemigrations-update but it only works with the latest migration.

Reply:
@SmittieC I think this can still be done which would reduce the migrations from 4 to 3. A workflow that I've used in the past is to undo the migrations and then have django re-make them:

```
# roll back to 73
./manage.py migrate experiments 0073

# remove the 2 migrations that need to be squashed
rm apps/experiments/migrations/0074*.py apps/experiments/migrations/0075*.py

# backup the later ones
mv apps/experiments/migrations/0076*.py apps/experiments/migrations/0077*.py apps/experiments/

# run django migrations to recreate initial model migrations
./manage.py makemigrations experiments

# move others back and rename
mv apps/experiments/0076_create_participants.py apps/experiments/migrations/0075_create_participants.py
mv apps/experiments/0077_create_participants.py apps/experiments/migrations/0076_create_participants.py

# edit the migrations to update their dependencies
```

Reply:
Ah yes, there is the 0074 migration as well, thanks. I just copied the operations and the migration's dependencies over to the previous one. I missed [this](https://github.com/dimagi/open-chat-studio/pull/363/commits/37f819dbab90964e6a0bd7e815007b546920cc51) one in [this](https://github.com/dimagi/open-chat-studio/pull/363/commits/cfbbc0ca2e81714bb2b51070ed67acc5a8f1faf0) commit.

Reply:
Hmm, actually I don't feel right about this move. Let me just do it the long way